x-pack/plugins/security/server/plugin.ts

```diff
@@ -307,6 +310,7 @@ export class SecurityPlugin
       getFeatureUsageService: this.getFeatureUsageService,
       getAuthenticationService: this.getAuthentication,
       getAnonymousAccessService: this.getAnonymousAccess,
+      usageCounter,
```

Comment:
Nit, we don't need to define this as a const above since this is only being used by the `defineRoutes` method:

```suggestion
      usageCounter: usageCollection?.createUsageCounter('security'),
```